pkg/k8shandler/curation.go

```diff
@@ -133,12 +133,12 @@ func createOrUpdateCuratorSecret(logging *logging.ClusterLogging) error {
 		"curator",
 		logging.Namespace,
 		map[string][]byte{
-			"ca":       utils.GetFileContents("/tmp/_working_dir/ca.crt"),
-			"key":      utils.GetFileContents("/tmp/_working_dir/system.logging.curator.key"),
-			"cert":     utils.GetFileContents("/tmp/_working_dir/system.logging.curator.crt"),
-			"ops-ca":   utils.GetFileContents("/tmp/_working_dir/ca.crt"),
-			"ops-key":  utils.GetFileContents("/tmp/_working_dir/system.logging.curator.key"),
-			"ops-cert": utils.GetFileContents("/tmp/_working_dir/system.logging.curator.crt"),
+			"ca":       utils.GetFileContents(utils.GetFilePath("ca.crt")),
```

Comment:
Why not abastract this further into a single util method for getting content from a known file location?

Reply:
Updated